common/app/views/support/GetClasses.scala

```diff
@@ -170,8 +169,9 @@ object GetClasses {
 
   def paletteClasses(container: Container, metadata: Seq[Metadata]): Option[Seq[String]] = {
     container match {
-      case Fixed(_) => primaryPaletteClass(metadata).map(Seq(_, "fc-container--has-palette"))
-      case _        => None
+      case Fixed(_) | Dynamic(DynamicSlow) | Dynamic(DynamicFast) | Dynamic(DynamicSlowMPU(_, _)) =>
```

Comment:
Could you do the invert, where Dynamic/package is None and `_` gets the class or is that not idomatic?

Reply:
Did that in my first commit but I didn't really want to target `nav/list`, `news/most-popular` or various other strange container types.

Reply:
Right, yeah, good shout 👍